zone/attack.cpp

```diff
@@ -4231,7 +4233,7 @@ void Mob::HealDamage(uint64 amount, Mob *caster, uint16 spell_id)
 						YOU_HEAL, GetCleanName(), itoa(acthealed));
 			}
 		}
-		else {
+		else if (this->CastToClient()->GetFilter(FilterHealOverTime) != (FilterShowSelfOnly || FilterHide)) {
```

Comment:
Remove `this->`

Reply:
I'm going to have to close this PR, I believe I had deleted the original branch. I've cloned it now and will submit a new PR with the change.